libtest.sh

```diff
@@ -0,0 +1,356 @@
+#!/usr/bin/env bash
```

Comment:
This is an old version without the comments.

Reply:
Please take the latest `libtest.sh` from `onet/master`.

Reply:
Arg, I don't know what happened here, thanks for catching this.